node/src/Unison/Runtime/Cryptography.hs

```diff
@@ -0,0 +1,85 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# Language ScopedTypeVariables #-}
+
+module Unison.Runtime.Cryptography where
+
+import Unison.Cryptography
+import Data.ByteString (ByteString)
+import Data.ByteArray as BA
+import qualified Crypto.Random as R
+
+-- cryptonite
+-- import Crypto.Cipher.Types.Base ()
+-- import Crypto.Cipher.AES.Primitive (AES)
+import qualified Crypto.Cipher.AES as AES
+import Crypto.Cipher.Types
+import Crypto.Error
+
+-- Creates a Unison.Cryptography object specialized to use the noise protocol
+-- (http://noiseprotocol.org/noise.html).
+mkCrypto :: forall cleartext symmetricKey . (ByteArrayAccess cleartext, ByteArray cleartext, ByteArrayAccess symmetricKey, ByteArray symmetricKey) => ByteString -> Cryptography ByteString symmetricKey () () () () cleartext
```

Comment:
Hmm, looking at this more you may be right in your suggestion that `AES256` is the key type but I hadn't thought so before.  I have a question out to the `cryptonite` folks to confirm.